### PR TITLE
docs: update roadmap and customer examples

### PR DESCRIPTION
**Pull request by herrardo**
Update Roadmap and customer examples

# Pull request template
 
Change the Roadmap section to adapt to the current status of the project and anticipate some features.
Change the customers because to be up to date with new customers.

## Motivation and context
Transparency

- [ ] Dependencies. If any, specify:
- [ ] Open issue. If applicable, link:

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Change requires a documentation update

## What is the destination branch of this PR?

- [x] `Main`
- [ ] Other. Specify:

## How has this been tested?

Tests performed according to [testing guidelines](./contributing/tests.md): 

## Checklist:

- [ ] My code follows the **[style guidelines](./CONTRIBUTING.md#style-guides)** of this project.
- [ ] I have performed a **self-review** on my own code.
- [ ] I have **commented** my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the **[documentation](./CONTRIBUTING.md#documentation-style-guide)**.
- [ ] My changes generate **no new warnings**.
- [ ] I have added **tests** that prove my fix is effective or that my feature works.
- [ ] New and existing **unit tests pass locally** with my changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,43 +82,55 @@ Empathy’s eDocs documentation portal.
...
-  [![Release Docs](https://img.shields.io/badge/In%20Progress-Q3%202022-yellow)](https://github.com/empathyco/x/tree/main/packages/x-tailwindcss)
-- WCAG AA compliance.
-- Result Preview component to rapidly discover more result attributed without leaving the SERP.
-- Suggestions with filters that allows to select a query suggestion with a specific filter.
-- Provide a No Network State for components depending on API requests.
-- Incorporate search box power-ups such as animated suggestions or forbidden character set to
-  prevent code injection.
-- Project generator CLI.
+- **Search box power-ups** such as
+  [animated suggestions](https://docs.empathy.co/develop-empathy-platform/ui-reference/components/search-box/x-components.search-input-placeholder.html)
```
Comment (javieri-empathy) [inline README.md:116]: This does not work on eDocs. Are they already aware?

File: README.md [hunk 2]
```diff
@@ -80,43 +82,55 @@ Empathy’s eDocs documentation portal.
...
-  after the search action. This represents Empathy sciences for **Inspiration** and
+- **Next Queries Preview**: A set of results that matches searches that other shoppers performed
+  after the current to be show within the SERP. This helps the shopper to discover interesting
+  products after the search action. This represents Empathy sciences for **Inspiration** and
   **Cross-Selling**.
   [![Release Docs](https://img.shields.io/badge/Released-August%202022-brightgreen)](https://docs.empathy.co)
-- Query Results Preview: Provides a list of results matching a query to be inserted before the
-  shopper starts typing as inspiration, in the predictive layer for results matching the current
-  query, or as inspiration in a no results scenario.
+- **[Query Results Preview](https://docs.empathy.co/develop-empathy-platform/build-search-ui/web-archetype-integration-guide.html#setting-up-the-queries-preview-dynamically)**:
```
Comment (javieri-empathy) [inline README.md:110]: Link is wrong (and there is a missing `>` in the script of that doc.)
```suggestion
- **[Query Results Preview](https://docs.empathy.co/develop-empathy-platform/build-search-ui/web-archetype-integration-guide.html#dynamic-query-results-preview)**:
```

**Review by javieri-empathy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (herrardo) [reply README.md:116]: Uhmm..is working as far as i see, could it be something temporal?

Comment (herrardo) [reply README.md:116]: Yes, the placeholder live example is not working, but it's not relative to this PR. Thanks for the heads up.

**Review by javieri-empathy — APPROVED** [review]

**Review by tiborux — APPROVED** [review]

**Pull request merged**